packages/strapi-plugin-users-permissions/controllers/Auth.js

```diff
@@ -310,6 +310,11 @@ module.exports = {
       );
     }
 
+    // User blocked
+    if (user.blocked) {
+      return ctx.badRequest('blocked.user');
```

Comment:
Can you use the same format as line 304? Thus the message can be directly displayed in the front!

(and if you can fix the missing "a" line 286 too :p)